src/util/sendEmail.ts

```diff
@@ -0,0 +1,68 @@
+import transporter from '../config/email'
+import { User } from '../components/users/user'
+
+interface Email {
+  subject: string
+  body: string
+  link?: string
+  linkTitle?: string
+}
+
+export const sendEmail = (recipients: User[], email: Email) => {
+  if (process.env.NODE_ENV === 'production') {
+    recipients.forEach(user => {
+      transporter.sendMail({
+        from: `TanulóSCH <${process.env.EMAIL_USER}>`,
+        to: user.email,
+        subject: email.subject,
+        text: email.body,
+        html: `<html>
```

Comment:
I'd move this HTML definition to a separate file (something like emailTemplate.ts) into a function that accepts the required parameters and returns the HTML.